src/ng-prettyjson.js

```diff
@@ -26,13 +26,18 @@ angular.module('ngPrettyJson', [])
       scope.settings = (scope.settings || {});
       scope.triggers = (scope.triggers || {});
       scope.edition = attrs.edition;
-      scope.aceEditor = window.ace !== undefined;    
+      scope.aceEditor = window.ace !== undefined;
+
+      scope.showEditButton = true;
+      if(scope.settings.showEditButton === false) {
+        scope.showEditButton = false;
+      }
```

Comment:
Actually, it looks like `isDefined` returns the value if it is defined; since this may be false, it returns false and that line sets `scope.showEditButton` to true when it should be false.